pkg/cosign/tuf/client.go

```diff
@@ -115,6 +116,7 @@ func New(ctx context.Context, remote client.RemoteStore, cacheRoot string) (*TUF
 	// Now check to see if it needs to be updated.
 	trustedTimestamp, ok := trustedMeta["timestamp.json"]
 	if ok && !isExpiredMetadata(trustedTimestamp) {
+		t.timestamp = trustedTimestamp
```

Comment:
I went back and forth on this when first implementing it. I've made the change now, and I think this actually looks nicer. We also can remove storing the `Close` function for the local store, since we're persisting the local store itself. Lemme know what you think!

Reply:
Is it fine to make a copy of LocalStore, or should the TUF struct be holding a pointer to it?

Reply:
LocalStore is an interface, so I don't think it should be a pointer (the impl can be a pointer) @dlorenc someone help me with Go hahha

Reply:
https://npf.io/2014/05/intro-to-go-interfaces/ - Says interfaces are just pointers, so then it seems like there's no reason to hold a pointer to it.

I need a primer on the memory model for golang :)